src/sympc/tensor/mpc_tensor.py

```diff
@@ -99,7 +99,6 @@ class MPCTensor(metaclass=SyMPCTensor):
         # If not, we could use the self.tensor requires_grad
         # Use for training
         "requires_grad",
-        "grad",
```

Comment:
Was this an unintentional change?

Reply:
There are two `grad` variables declared.It was a typo in previous PR.

Reply:
Oh okay.